pynest/nest/tests/test_aeif_lsodar.py

```diff
@@ -281,11 +281,11 @@ def test_closeness_nest_lsodar(self):
         self.assert_pass_tolerance(rel_diff, di_tolerances_lsodar)
 
     @unittest.skipIf(not HAVE_GSL, 'GSL is not available')
-    def test_iaf_behaviour(self):
-        '''
-        The models should behave as iaf_cond_* if a == 0., b == 0. and
-        Delta_T == 0.
-        '''
+    def testIafBehaviour(self):
+
+        # The models should behave as iaf_cond_* if a == 0., b == 0. and
+        # Delta_T == 0.
```

Comment:
@heplesser I might be wrong but I think the problem had nothing to do with line breaks contained in the docstring. Nest's outpout to stdout/stderr just simply comes in between, the report on the unit test printed by nosetests and the "... ok" string at the end of single test run. The line breaks came from the Nest output. 
If the first line of the docstring would contain a meaningful test description, things are perfect. We have > 500 nose-based unit tests (I believe) where most of it do not make use of a docstring. There is some effort adding docstrings to all tests. 
.. but yes, let's discuss this in the VC.